core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -109,6 +115,10 @@ oppia.directive('explorationSummaryTile', [
             $scope.explorationIsCurrentlyHoveredOver = hoverState;
           };
 
+          $scope.loadParentExploration = function() {
+            $window.open($scope.getExplorationLink(), '_self');
```

Comment:
Just curious... Any specific reason for using .open() over $window.location.href = url? SO says the href approach is preferred ([link](https://stackoverflow.com/questions/4813879/window-open-target-self-v-window-location-href)].

Reply:
Done, changed it. Also, no, there was no particular reason for that.